pkgs/top-level/all-packages.nix

```diff
@@ -27348,6 +27361,9 @@ in
 
   wrapFirefox = callPackage ../applications/networking/browsers/firefox/wrapper.nix { };
 
+  # Only used for new Thunderbird 91.x
+  wrapThunderbird = callPackage ../applications/networking/mailreaders/thunderbird/91/wrapper.nix { };
```

Comment:
Is there prior art into naming wrappers ?
I'm just wondering if `gpgWrapThunderbird` or something more indicative of what it wraps it with would be clearer for outsiders ?
Just a small thought, no strong feelings here.

Reply:
(since this is a backport, it's probably not needed in this PR, just wanted to bring this up)

Reply:
`gpgWrapThunderbird` would be a very misleading name, since it also does the general wrapping with gtk libs etc. by calling the firefox wrapper